models/gmx/avalanche_c/glp/gmx_avalanche_c_glp_schema.yml

```diff
@@ -0,0 +1,196 @@
+        tests:
+          - not_null
+      - &wbtc_e_current_price
+        name: wbtc_e_current_price
+        description: "Current price of WBTC.e as calculated per the GMX smart contract"
+        tests:
+          - not_null
+      - &wbtc_e_longs
+        name: wbtc_e_longs
+        description: "Net value of WBTC.e longs in USDG"
+        tests:
+          - not_null
+      - &wbtc_e_shorts_entry_price
+        name: wbtc_e_shorts_entry_price
+        description: "Average price (volume weighted) of all WBTC.e shorts"
+        tests:
+          - not_null
+      - &wbtc_e_shorts_outstanding_notional
+        name: wbtc_e_shorts_outstanding_notional
+        description: "Sum of all WBTC.e shorts in USDG"
+        tests:
+          - not_null
+      - &btc_b_available_assets
+        name: btc_b_available_assets
+        description: "BTC.b tokens that are available for taking long positions against the pool, calculated as poolAmounts minus reservedAmounts"
+        tests:
+          - not_null
+      - &btc_b_current_price
+        name: btc_b_current_price
+        description: "Current price of BTC.b as calculated per the GMX smart contract"
+        tests:
+          - not_null
+      - &btc_b_longs
+        name: btc_b_longs
+        description: "Net value of BTC.b longs in USDG"
+        tests:
+          - not_null
+      - &btc_b_shorts_entry_price
+        name: btc_b_shorts_entry_price
+        description: "Average price (volume weighted) of all BTC.b shorts"
+        tests:
+          - not_null
+      - &wbtc_b_shorts_outstanding_notional
+        name: btc_b_shorts_outstanding_notional
+        description: "Sum of all BTC.b shorts in USDG"
+        tests:
+          - not_null
+
+  - name: gmx_avalanche_c_glp_aum
+    meta:
+      blockchain: avalanche_c
+      project: gmx
+      contributors: theachenyj
+    config:
+      tags: ['avalanche_c','gmx','glp']
+    description: >
+        View GLP net AUM by pools.
+    columns:
+      - *minute
+      - &usdc_aum
+        name: usdc_aum
+        description: "Net asset under management for the USDC pool"
+        tests:
+          - not_null
+      - &usdc_e_aum
+        name: usdc_e_aum
+        description: "Net asset under management for the USDC.e pool"
+        tests:
+          - not_null
+      - &mim_aum
+        name: mim_aum
+        description: "Net asset under management for the MIM pool"
+        tests:
+          - not_null
+      - &wavax_aum
+        name: wavax_aum
+        description: "Net asset under management for the WAVAX pool"
+        tests:
+          - not_null
+      - &weth_e_aum
+        name: weth_e_aum
+        description: "Net asset under management for the WETH.e pool"
+        tests:
+          - not_null
+      - &wbtc_e_aum
+        name: wbtc_e_aum
+        description: "Net asset under management for the WBTC.e pool"
+        tests:
+          - not_null
+      - &btc_b_aum
+        name: wbtc_aum
```

Comment:
Hi @theachenyj, you may have a typo here, do you mean `btc_b_aum`.

Reply:
Nice catch! I'd committed a fix. Thx.